### DISCUSSION TITLE
Dark theme

### DISCUSSION THREAD

**Discussion by rperson1508** [executablebooks/meta#745] [Q&A]
Hello, how do I configure dark mode for sphinx-book-theme?
I have not found any specific way to do so in the documentation.

**Comment by choldgraf** [comment]
It does not currently exist, see this GitHub issue for discussion of adding this enhancement:

- https://github.com/executablebooks/sphinx-book-theme/issues/265

**Reply by rperson1508** [reply]
Alright but I thought I saw that it has been merged?

**Comment by rperson1508** [comment]
Digged further and found the [customizing guide](https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/customizing.html) on the docs.

#### conf.py
```py
html_theme = 'sphinx_book_theme'
html_static_path = ['_static']
html_css_files = [
    'css/custom.css'
]
```

#### _static/css/custom.css
```css
/* Dark theme */
html[data-theme="dark"] {
    background: #000000;
}
```

After re-building the documentation (and refreshed), the background colour did not change. Were there any mistakes I made?
